desktop/src/main/java/haveno/desktop/theme-dark.css

```diff
@@ -9,17 +9,17 @@
     -fx-text-fill: #dadada;
 
     /* javafx elements */
-    -fx-accent: #25b135;
+    -fx-accent: #0b65da;
     -fx-box-border: transparent;
-    -fx-focus-color: #2ea33c;
-    -fx-faint-focus-color: #2ea33c;
-    -fx-selection-bar: #1a6e1e;
-    -fx-selection-bar-non-focused: #2a7e2e;
+    -fx-focus-color: -fx-accent;
```

Comment:
Thanks for your contribution. :) Overall it works.

I would expect consistency with Bisq's highlighting scheme.

In this case, focus-color, faint-focus-color, and color-primary-dark would have the same color.

And selection-bar and selection-bar-non-focused would have different colors.

Is the highlighting scheme changed intentionally, or should this be adjusted?

Were the highlighting